spec/rubocop/cop/layout/else_alignment_spec.rb

```diff
@@ -60,18 +60,15 @@
               c
             end
         RUBY
-        expect(cop.messages).to eq(['Align `elsif` with `if`.',
-                                    'Align `else` with `if`.'])
-        expect(corrected)
-          .to eq <<-RUBY.strip_margin('|')
-            |  if a1
-            |    b1
-            |  elsif a2
-            |    b2
-            |  else
-            |    c
-            |  end
-          RUBY
+        expect(corrected).to eq(<<-RUBY)
+            if a1
```

Comment:
The indent here looks wrongs. `strip_indent` is also missing.

Reply:
In this case the cop shifts the `elsif` to the right, instead of everything else to the left. The indentation is then supposedly handled by another cop.

Reply:
I just have this habit to indent the code in all samples as they'd be seen in a source file. Not a biggie, though.

Reply:
I'm wondering if `#strip_margin` would actually be clearer in the case of some of the examples. Will think about it.

Reply:
If they feature non-standard indentation from line 1 - probably yes.